charts/datadog/values.yaml

```diff
@@ -691,6 +691,12 @@ datadog:
      ## ref: https://docs.datadoghq.com/infrastructure/containers/?tab=helm
     containerCollection: true
 
+    # datadog.processAgent.disableOsReleaseFileMount -- Set this to true to disable mounting datadog.osReleasePath in processAgent container
+    disableOsReleaseFileMount: false
```

Comment:
As, I mentioned in a previous comment, we have already a similar option but only for system-probe container (`datadog.systemProbe.enableDefaultOsReleasePaths`).

Could you add `datadog.disableDefaultOsReleasePaths: false`  instead of `datadog.processAgent.disableOsReleaseFileMount ` and deprecate `datadog.systemProbe.enableDefaultOsReleasePaths`?

By deprecation I mean: continue to support `datadog.systemProbe.enableDefaultOsReleasePaths` in the chart but flag it as deprecated. we will remove it later in a major version. 

for example here: https://github.com/DataDog/helm-charts/blob/3dc5c377cd05744a512b147d72a0b25e8631fca1/charts/datadog/templates/_container-system-probe.yaml#L68
replace 
``` yaml
{{- if .Values.datadog.systemProbe.enableDefaultOsReleasePaths }}
```
by 
```yaml
{{- if or .Values.datadog.systemProbe.enableDefaultOsReleasePaths (not .Values.datadog.disableDefaultOsReleasePaths) }}
```
So the new setting can have an impact everywhere, but we continue to support the previous option

Reply:
sorry for the delay, the tests are ongoing on my side. I will update the PR ASAP

Reply:
updated too :)